integration-tests/kamelet/src/main/resources/application.properties

```diff
@@ -15,3 +15,8 @@
 ## limitations under the License.
 ## ---------------------------------------------------------------------------
 camel.kamelet.setBodyFromProperties.bodyValueFromProperty=Camel Quarkus Kamelet Property
+
+quarkus.camel.kamelet.identifiers = injector,logger
+
+# this is needed to actually test that kameleta are preloaded at build time:
+camel.component.kamelet.location = file:/invalid
```

Comment:
Is this just for test or do users need to have this attribute? 
Can we choose another file location ?

Reply:
That is to be sure that test at runtime uses the preloaded kamelets.
Because if it tried to reload the kamelet there would be an error (due to the invalid path).

Reply:
Ok thanks.